optionals/compat_sog/CfgVehicles.hpp

```diff
@@ -20,4 +20,6 @@ class CfgVehicles {
     #include "CfgVehicles\turrets.hpp"
     #include "CfgVehicles\units.hpp"
     #include "CfgVehicles\wheeled.hpp"
+    #include "cfgVehicles\land.hpp"
+    #include "cfgVehicles\vn_boxes.hpp"
```

Comment:
Best compiler is case sensitive :)
```suggestion
    #include "CfgVehicles\land.hpp"
    #include "CfgVehicles\vn_boxes.hpp"
```